app/retail/templates/emails/gdpr_update.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'emails/template.html' %}
+{% comment %}
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+{% load i18n humanize %}
+{% load static %}
+
+{% block content %}
+
+<h1>{% trans "Gitcoin: Updated Terms & Policies" %}</h1>
+<div style="text-align: left! important">
+    <p>
+        {% trans "Hello" %} {{subscriber}},
+    </p>
+    <p>
+        {% trans "As a Web 3.0 company, we think carefully about user data and privacy and how the internet is evolving. We hope Web 3.0 will bring more control of data to users. With this ethos, we are always careful about how we use your information. " %}
+    </p>
+    <p>
+        {% trans "We recently reviewed our Privacy Policy to comply with requirements of General Data Protection Regulation (GDPR), improving our Terms of Use, Privacy Policy and Cookie Policy. These changes will go into effect on May 25, 2018, and your continued use of the Gitcoin after May 25, 2018 will be subject to our updated Terms of Use and Privacy Policy. " %}
+    </p>
+    <p>
+        {% trans "As always, the entire Gitcoin team works every day to make Gitcoin a safe and trustworthy place to grow open source. We invite you to learn more about our outlined updates to our Policies and Terms of Use: " %}
+    </p>
+    <ul>
+        <li>{% blocktrans %}<a href="{{terms_of_use_link}}">Our updated Terms of Use </a>{% endblocktrans %}</li>
```

Comment:
To avoid issues with autogeneration of translations, I've been trying to avoid wrapping dynamic vars in `blocktrans`.  We could update this to:
`<li><a href="{{ terms_of_use_link }}">{% trans "Our updated Terms of Use" %} </a></li>`